### PR TITLE
Header section

### PR DESCRIPTION
**Pull request by willy836**
This project contains the header section (toolbar and headline) of the portfolio mobile first website

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+                <p class="text">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+                <p class="connect">LET'S CONNECT</p>
+                <div class="social-icons">
+                    <img class="img1" src="images/twitter.png" alt="twitter image">
+                    <img class="img2" src="images/linked1.png" alt="linkedin image">
+                    <img class="img6" src="images/linked2.png" alt="linkedin image">
+                    <img class="img7" src="images/linked3.png" alt="linkedin image">
+                    <img class="img3" src="images/Vector_M.png" alt="vector">
+                    <img class="img4" src="images/github.png" alt="github image">
+                    <img class="img5" src="images/chill.png" alt="vector">
```
Comment (CollinsTatang) [inline index.html:29]: - [ ] Please wrap the social links in a `ul` tag to group them, and `li` per icon; each icon has a link (a tag)

**Review by CollinsTatang — CHANGES_REQUESTED** [review]
Hi @willy836, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

### **Highlights 🚀 :heavy_check_mark: :heavy_check_mark:** 

- The repo looks professional.:heavy_check_mark:
- Git/GitHub flow best practices have been followed:heavy_check_mark:
- There are no linter errors:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:

## Required changes 
It's ok to have bugs in your code. Please, see this code review as a chance for you to learn practical coding skills and learn new tips that can improve your code quality and help you get a better job.
_Check the comments under the review._
## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @CollinsTatang in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="https://kit.fontawesome.com/928599f9b9.js" crossorigin="anonymous"></script>
+</head>
+<body>
```
Comment (Munsa1) [inline index.html:11]: - [ ] The whole point of this project is to follow the given Figma template as much as possible, this includes background images, fonts, colors, and spacings. Kindly add the provided background images and colors to your design.

Your Design
image

Figma Design
image

- I would suggest you fix the colors for the text and social icons as well as add the background images using CSS and not in the html file.

**Review by Munsa1 — CHANGES_REQUESTED** [review]
Hi @willy836, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
:heavy_check_mark: Descriptive Readme
:heavy_check_mark: Passing Linters
:heavy_check_mark: Correct Gitflow

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+    <section id="header">
+        <div class="toobar-container">
+            <nav id="nav">
+                <div class="logo">My Logo</div>
+                <div class="menu"><i class="fa-solid fa-bars"></i></div>
+            </nav>
```
[the remark this answers is not in the document]
Comment (CollinsTatang) [reply index.html]: Hi @willy836 
Sorry for the typo here it was to `header` and not `section`

**Review by codecaiine — APPROVED** [review]
Hi @willy836 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- Clean code :heavy_check_mark: 
- Good project implementation :heavy_check_mark: 
- Great request changes application :heavy_check_mark: 
- Description PR :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me (@codecaiine) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**